### ISSUE TITLE
3D EM Map, 3D EM Mask : position in hierarchy?

### ISSUE THREAD

**Issue by joncison** [edamontology/edamontology#382]
@jlgelpi 

Currently [3D EM Map](http://edamontology.org/data_3805) and [3D EM Mask](http://edamontology.org/data_3806) are currently kids of [Structure](http://edamontology.org/data_0883), which doesn't seem right.

Can you please advise - do we need a new parent concept, or are they OK where they are ?

**Comment by jlgelpi** [comment]
I agree, they are no "Structure". They could fit under "image", as they are secondary data from EM images, but I would prefer to have a whole data section on experimental data leading to 3D structure, together with X-Ray diffraction maps, Structure Factors, NMR experimental data, that are also missing. We need a more expert advice here. 
+ juergen.haas@unibas.ch
+ carazo@cnb.csic.es

**Comment by joncison** [comment]
ok thanks

so I leave this open till we hear from others, we already have:

[Experimental measurement](http://edamontology.org/data_3108) -> [Protein structure raw data](http://edamontology.org/data_2537) and various kids of that (prob. needs cleaning!)

**Comment by joncison** [comment]
Hi @jlgelpi could I get a bit more advice here?  Also from @veitveit (re NME experimental data etc.)  thanks!

**Comment by veitveit** [comment]
There is probably more as new techniques are emerging. What about raw data from CryoEM?

I am not an expert at all here, so we would need help.
